sycl/source/detail/memory_manager.cpp

```diff
@@ -315,32 +341,30 @@ void MemoryManager::copy(SYCLMemObjT *SYCLMemObj, void *SrcMem,
                          unsigned int DimDst, sycl::range<3> DstSize,
                          sycl::range<3> DstAccessRange, sycl::id<3> DstOffset,
                          unsigned int DstElemSize,
-                         std::vector<cl_event> DepEvents,
-                         bool UseExclusiveQueue, cl_event &OutEvent) {
-
+                         std::vector<cl_event> DepEvents, cl_event &OutEvent) {
   if (SrcQueue->is_host()) {
     if (TgtQueue->is_host())
       copyH2H(SYCLMemObj, (char *)SrcMem, std::move(SrcQueue), DimSrc, SrcSize,
               SrcAccessRange, SrcOffset, SrcElemSize, (char *)DstMem,
               std::move(TgtQueue), DimDst, DstSize, DstAccessRange, DstOffset,
-              DstElemSize, std::move(DepEvents), UseExclusiveQueue, OutEvent);
```

Comment:
Same request as prior comment, not too sure why we're removing UseExclusiveQueue